### ISSUE TITLE
Is @types/terser-webpack-plugin@5.2.0 outdated?

### ISSUE THREAD

**Issue by PabloLION** [electron-react-boilerplate/electron-react-boilerplate#3064]
## Summary

Hi team. I just started a new electron-react project.
during the installation, I see some warnings:

```node
warning @types/terser-webpack-plugin@5.2.0: This is a stub types definition. terser-webpack-plugin provides its own type definitions, so you do not need this installed.
warning webpack-dev-server > url > querystring@0.2.0: The querystring API is considered Legacy. new code should use the URLSearchParams API instead.
[2/4] 🚚  Fetching packages...
[3/4] 🔗  Linking dependencies...
warning " > @typescript-eslint/eslint-plugin@5.7.0" has unmet peer dependency "@typescript-eslint/parser@^5.0.0".
warning " > enzyme-adapter-react-16@1.15.6" has incorrect peer dependency "react@^16.0.0-0".
warning " > enzyme-adapter-react-16@1.15.6" has incorrect peer dependency "react-dom@^16.0.0-0".
warning "enzyme-adapter-react-16 > enzyme-adapter-utils@1.14.0" has incorrect peer dependency "react@0.13.x || 0.14.x || ^15.0.0-0 || ^16.0.0-0".
warning "enzyme-adapter-react-16 > react-test-renderer@16.14.0" has incorrect peer dependency "react@^16.14.0".
warning "enzyme-adapter-react-16 > enzyme-adapter-utils > airbnb-prop-types@2.16.0" has incorrect peer dependency "react@^0.14 || ^15.0.0 || ^16.0.0-alpha".
```

What I want to ask is that if `@types/terser-webpack-plugin@5.2.0` is no longer needed? Should I make a PR removing it?

**Comment by amilajack** [comment]
Fixed in dc822dc212a37862cbf5e293cc5122ed954079f7

**Comment by PabloLION** [comment]
Thank you
